### ISSUE TITLE
initial plan

### ISSUE THREAD

**Issue by drphilmarshall** [drphilmarshall/FuzzyBlobs#4]
@dsand - How's it going? Did you and Paul make a plan to pick this project up and run with it? Let us know!

**Comment by dsand** [comment]
Yes, Paul is a little ethereal.  So if he doesn't do it, I will (on slower time scales, as I'm teaching). 
More soon.
